tasks/helm_install_chart.rb

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env ruby
+
+require 'tmpdir'
+require 'yaml'
+require_relative '../files/pam_task_helper.rb'
+
+# Install or upgrade a Helm chart.
+class HelmInstallChart < PAMTaskHelper
+  def task(chart:, release:, namespace:, kubeconfig:, version: nil, values: nil, **_kwargs)
+    kubeconfig_path = File.expand_path(kubeconfig)
+
+    install_command = [
+      'helm',
+      'upgrade',
+      release,
+      chart,
+      '--install',
+      "--namespace=#{namespace}",
+      "--kubeconfig=#{kubeconfig_path}",
```

Comment:
Has specifiying the kubeconfig path been necessary somewhere? Often it's simpler to let the tool find config, and use the KUBECONFIG environment variable to override it.

Reply:
KOTS also uses KUBECONFIG, but we're not explicitly specifying `--kubeconfig` there.

Reply:
We're going to need to start allowing override of kubeconfig to provide for executing against arbitrary configs (to gke clusters, for example), or for any case where the env that bolt is executing in doesn't have a default kubeconfig that is the one we want. I don't think we have a mechanism for setting KUBECONFIG for task execution, unless possibly we switch the tasks to take input from environment variables instead of stdin.

I have touched any of the other tasks yet, or looked this problem over more generally, but I wanted to plumb the new task at least.

Reply:
It might end up being sufficient to say that whatever the test runner is, it needs to have been configured with a KUBECONFIG that will work for the intended operation.

Reply:
I could see this being needed in something more general, like setting up a GKE instance then installing the application. Although some methods of setting up a pod will also update the local kube context.